this & object prototypes/ch1.md

```diff
@@ -40,11 +40,11 @@ speak.call( me ); // Hello, I'm KYLE
 speak.call( you ); // Hello, I'm READER
 ```
 
-If the *how* of this snippet confuses you, don't worry! We'll get to that shortly. Just set those questions aside briefly so we can look into the *why* more clearly.
+Se o *como* desse snippet confunde você, não se preocupe! Já vamos chegar lá. Apenas coloque isso de lado por um momento para que possamos olhar mais claramente o *porquê*.
```

Comment:
No caso no snippet, seria legal traduzi-lo, como referenciado no [wordreference](https://github.com/cezaraugusto/You-Dont-Know-JS/blob/portuguese-translation/WORDREFERENCE.md)

É um termo bem comum em dev, mas algumas pessoas podem não conhecer, principalmente iniciantes.

```diff
- Se o *como* desse snippet confunde você
+ Se o *como* desse trecho de código confunde você
```